<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>1060</width>
      <height>820</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_windowlevel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;窗位：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Flip_vertical">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>垂直翻转</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Save">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>保存图片</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rote">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;旋转：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_windowLevel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>105</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>4096</number>
    </property>
    <property name="pageStep">
     <number>100</number>
    </property>
    <property name="value">
     <number>2048</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_scaling">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>205</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Gray_inverse">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>451</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>灰度反转</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rest_Flip">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>360</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Open">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>打开RAW</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_scaling">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;放缩：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Flip_horizontal">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>水平翻转</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Flip_diagonal">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>360</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>对角翻转</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_windowWidth">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>155</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>4096</number>
    </property>
    <property name="pageStep">
     <number>100</number>
    </property>
    <property name="value">
     <number>4096</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_windowwidth">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;窗宽：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_rote">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-180</number>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Enhance">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>锐化</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_windowLevel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>105</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>2048</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_windowWidth">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>155</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>4096</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_scaling">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>205</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_rote">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>255</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reset_wL">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>100</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reset_wW">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reset_S">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reset_R">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Origin">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>原始图像</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_log">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>520</y>
      <width>400</width>
      <height>340</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>74</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>33</number>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_Current">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>520</y>
      <width>40</width>
      <height>110</height>
     </rect>
    </property>
    <property name="text">
     <string>数
据
存
档</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_log">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>470</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;操作日志&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_double_click">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>630</y>
      <width>40</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;双&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;击&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;读&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;取&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Clear_log">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>760</y>
      <width>40</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>全
部
清
除</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_enhance">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>305</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_enhance">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rest_enhance">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>300</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
